res/values/strings.xml

```diff
@@ -859,12 +876,15 @@
     <string name="notify_dismiss">Dismiss</string>
     <string name="notify_reply_button">Reply</string>
     <string name="notify_new_message">New message</string>
+    <string name="notify_new_messages">New messages</string>
     <string name="notify_background_connection_enabled">Background connection enabled</string>
     <string name="notify_priority_high">High</string>
     <string name="notify_priority_max">Max</string>
     <string name="notify_name_and_message">Name and message</string>
     <string name="notify_name_only">Name only</string>
     <string name="notify_no_name_or_message">No name or message</string>
+    <string name="notify_bundle_new_messages_in_one_chat">%1$d new messages</string>
```

Comment:
"%1$d new messages" already exists as `chat_n_new_messages` and also handles plural forms.

Reply:
right, though only plural is needed in this case, because single messages are displayed on as dedicated notifications.